src/js/components/AppStatusComponent.jsx

```diff
@@ -59,11 +59,8 @@ var AppStatusComponent = React.createClass({
   render: function () {
     var model = this.props.model;
 
-    var statusClassSet = classNames({
-      "app-status": true,
-      "text-warning": this.isWarningStatus(),
-      "text-danger": model.status === AppStatus.DELAYED
-    });
+    var statusClassSet = classNames("app-status",
+      statusNameMapping[model.status].toLowerCase());
```

Comment:
Yes, this is also an option.
This would make the generated markup more descriptive, not the code itself, not needed imho, but if you like it, feel free to do it.